### ISSUE TITLE
Training iteration

### ISSUE THREAD

**Issue by ZYJ-JMF** [clovaai/mxfont#9]
HI, thanks for your impressive work.
I am trying training your code from the begining, so I was wondering how many GPUS you used and how long will it spends for training?

I noticed that you set'max_iter' to 800,000 and batch_size is 8. But in paper you said the iter is 650,000 and mini_batch size is 24.
So which is correct?

When I set 'batch size' to 24, can I reduce the iteration to 650k/3 ? can I get the same result?
I am looking forward to your response.Thanks

**Comment by 8uos** [comment]
Hi, thanks for your interest.

1) I trained the model for 10 days with a single V100 GPU.

2) Iteration: 650,000 is right.

3) Batch size: `batch_size` in this code means the number of images to generate at once. 
As our model extracts style (or content) from `n_in_s` (or `n_in_c`) images sharing the same style (or content), the model will actually take `batch_size` * `n_in_s` images at once. 
We used 8 for the `batch_size` and 3 for the `n_in_s` and `n_in_c`, and "24" in the paper means this.

4) I cannot sure that using batch size=24 and iteration=650k/3 will produce the same results as ours.
I have observed that increasing the batch size 4 to 8 enhances the results.
However, it does not mean that it is okay to reduce the number of iterations to half; for example, the results at 600k and batch_size=4 were better than the results at 300k and batch_size=8.
So, if you want to reproduce our results, I recommend using our hyperparameter settings.
